npm.js

```diff
@@ -62,12 +62,16 @@ exports.translate = function(load){
 				packages[pkg.name+"@"+pkg.version] = true;
 			}
 		});
-		var configDependencies = ['@loader','npm-extension'].concat(configDeps.call(loader, pkg));
+		var configDependencies = [
+			'@loader',
+			'npm-extension',
+			'process'
```

Comment:
This won't work, these dependencies load before the npm extension has loaded.  Since that process module needs to be required I'm not sure what the best solution is.  Maybe we just keep incrementally adding to our global process as we find needs.